.devcontainer/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd /home/vscode/catkin_ws/src;
+./mushr/mushr_utils/install/mushr_install.bash --trivial-only;
```

Comment:
By trivial, I answering "no" to all prompted questions. The use of word "trivial" was inspired by how `apt` uses that word. See https://linux.die.net/man/8/apt-get
> --trivial-only
> Only perform operations that are "trivial". Logically this can be considered related to --assume-yes. Where --assume-yes will answer yes to any prompt, --trivial-only will answer no.

I was tempted to use `--assume-no` but I haven't seen any other linux command using such wording, but happy to be wrong here and use something else.

Reply:
`assume-no` or `no-to-all` seems more informative. Trivial to me seems vague